chutney/server/src/main/java/com/chutneytesting/ServerConfiguration.java

```diff
@@ -237,6 +241,16 @@ CampaignService campaignService(CampaignExecutionRepository campaignExecutionRep
         return new CampaignService(campaignExecutionRepository);
     }
 
+    @Bean
+    CampaignEnvironmentUpdateHandler campaignEnvironmentUpdateHandler(CampaignRepository campaignRepository) {
+        return new CampaignEnvironmentUpdateHandler(campaignRepository);
+    }
+
+
+    @Bean
+    EnvironmentConfiguration environmentConfiguration(@Value(ENVIRONMENT_CONFIGURATION_FOLDER) String storeFolderPath, List<UpdateEnvironmentHandler> updateEnvironmentHandlers) {
```

Comment:
Where is used this bean ? isn't instantiated in environment module Spring configuration ?

Reply:
no check bean name. 
May be add an explanation in comment